data/json/recipes/armor/storage.json

```
@@ -982,5 +982,18 @@
     "autolearn": true,
     "using": [ [ "sewing_standard", 5 ] ],
     "components": [ [ [ "fabric_standard", 3, "LIST" ], [ "fabric_hides_any", 3, "LIST" ] ] ]
+  },
+  {
+    "result": "bandolier_grenade",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "skills_required": [ [ "gun", 1 ], [ "launcher", 1 ] ],
+    "time": "1 h 20 m",
```

```suggestion
    "time": "1 h",
```
A bit long, 1 hour is fine too

---

1 hour 20 minutes is the same as all other bandolier recipes in the json. It's not very long either given what you're sewing together.